handout/a11y/key-concerns/semantic-markup.md

```diff
@@ -0,0 +1,166 @@
+# Semantic Markup
+
+## Using Proper HTML Elements and Attributes
+
+A common trap when structuring html is using too many divs and marking them up with classes or ids to indicate their role, for example:
+
+```
+@Component({
+  selector: 'ngc2-app'
+  template: `
+    <div class="header">
+      <div class="navigation">
+        <div class="item"><a [routerLink]="['']"><img src="https://angular.io/resources/images/logos/angular2/angular.svg" width="40"></a></div>
+        <div class="item"><a [routerLink]="['services']">Services</a></div>
+        <div class="item"><a [routerLink]="['process']">Process</a></div>
+        <div class="item"><a [routerLink]="['work']">Work</a></div>
+      </div>
+    </div>
+    <router-outlet class="page-content">
+    </router-outlet>
+  `,
+})
+```
+[View Example](http://plnkr.co/edit/cm3wBDRqIrmxpECiQhg7?p=info)
+
+While it might be obvious to someone reading the html what the purpose of each element is, or someone viewing the site, the class names don't have any semantic meaning for browsers and screen readers. We can give them more information by using the proper html elements instead.
+
+```
+@Component({
+  selector: 'ngc2-app',
+  template: `
+    <header>
```

Comment:
Any consideration for landmarks here? 

https://www.w3.org/WAI/GL/wiki/Using_ARIA_landmarks_to_identify_regions_of_a_page